doc/item_bonus.txt

```diff
@@ -274,7 +276,6 @@ Ignore Def
 ----------
 bonus bIgnoreDefEle,e;  		Disregard DEF against enemies of element e
 bonus bIgnoreDefRace,r; 		Disregard DEF against enemies of race r
-bonus2 bIgnoreDefRaceRate,r,n;  	Disregard n% of the target's DEF if the target belongs to race r
```

Comment:
Why remove this? It's defined as a valid item bonus:
```cpp
export_constant2("bIgnoreDefRaceRate",SP_IGNORE_DEF_RACE_RATE);
```

Reply:
It's duplicated.
![image](https://user-images.githubusercontent.com/9493525/78091461-11bdf700-73f7-11ea-9688-8eca65b455f8.png)